openfisca_us/variables/gov/states/il/tax/income/credits/property_tax/il_property_tax.py

```diff
@@ -0,0 +1,10 @@
+from openfisca_us.model_api import *
+
+
+class il_property_tax(Variable):
```

Comment:
Move this to alongside `real_estate_taxes` and call it `property_tax_primary_residence` with a formula equal to `real_estate_taxes`